src/sparseml/pytorch/utils/exporter.py

```diff
@@ -461,7 +476,6 @@ def export_onnx(
     create_parent_dirs(file_path)
 
     module = deepcopy(module).cpu()
-    module.eval()
```

Comment:
justification?

Reply:
Let me look back into if this is needed anymore

Reply:
This was related to fusing of conv/bn during eval. keeping this in as all testing was done with this